extra-dev/packages/coq-coqffi/coq-coqffi.dev/opam

```diff
@@ -17,10 +17,10 @@ build: [
   ["dune" "build" "-p" name "-j" jobs]
 ]
 depends: [
-  "ocaml" {>= "4.08" & < "4.12~" }
+  "ocaml" {>= "4.08" & < "5.0~" }
```

Comment:
@lthms if your OCaml code is fundamentally incompatible with OCaml 5, the preferred way to state this in opam packages is as follows:
```
conflicts: [
  "base-nnp"
  "ocaml-option-nnpchecker"
]
```

Reply:
I don’t think it is fundamentally incompatible, I just didn’t try/adapt it yet.